_config.yaml

```diff
@@ -88,6 +88,7 @@ defaults:
 
 geolexica:
   concepts_glob: "../isotc204-glossary/concepts/*.yaml"
+  images_glob: "../isotc204-glossary/images/*.png"
```

Comment:
While this is fine for now, I want a stable container format for a concept dataset. i.e. all concept files are located within `concepts/`, all images are within `images/`, etc. and there is no need to specify any "glob pattern". The concept dataset needs to be a "defined format".

Reply:
@ronaldtse I've updated the flow to use the `glossary_path` to generate images path. It works by adding `/images` at the end of `glossary_path`.